entries/normalizer.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
+	<longdesc>
+		The user can change the value of an element before validating the element in question. The new value will be then used by the associated methods instead of the real one.
```

Comment:
Long-desc should contain
- normalizer doesnt change the elements value
- normalizer gets the value passed as argument
- normalizer needs to return a string
- "this" within a normalizer references the corresponding dom-element